package.json

```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "https://github.com/eciis/webchat#readme",
   "dependencies": {
+    "ip": "^1.1.5",
```

Comment:
I don't think we need this ip dependency. We *have* to know our address, and it's better if we set our hostname through a real domain name (rather than an IP).